tests/components/roborock/snapshots/test_vacuum.ambr

```diff
@@ -0,0 +1,24 @@
+# serializer version: 1
+# name: test_get_maps
+  dict({
+    'vacuum.roborock_s7_maxv': dict({
+      'maps': list([
+        dict({
+          'flag': 0,
```

Comment:
Is there any use for this for an end-user or can it be removed (I see no use for it)?

Reply:
It is used for
load_multi_map
name_multi_map
recover_multi_map

which are used by some end users.

Reply:
I could go either way on this.